tcms/testruns/static/testruns/js/get.js

```diff
@@ -282,11 +285,16 @@ function addLinkToExecution(event, testExecution) {
 function renderLink(link) {
     const linkEntryTemplate = $('#link-entry')[0].content
     const template = $(linkEntryTemplate.cloneNode(true))
+    const linkUrlEl = template.find('.link-url')
     if (link.is_defect) {
-        template.find('.link-icon').addClass('fa fa-bug')
+        linkUrlEl.addClass('fa fa-bug')
```

Comment:
The previous alternative is better IMO b/c it places the bug icon inside the <span> tag preceding the URL, which makes the icon black (default text color) and has some margin between the icon and the text.

With the current implementation the icon is blue and there is absolutely no margin. Doesn't look good IMO but also renders the link-icon span element redundant.